### ISSUE TITLE
[Bug] Possible dead path in Photographic

### ISSUE THREAD

**Issue by Dwscdv3** [Ben987/Bondage-College#93]
## TL;DR

**While the employee's arms are bound, she's able to bind you (which shouldn't be) but unable to release you.**

## Reproduce

1. Tie up the employee's arms and optionally other body parts but not the mouth.
2. Ask her to tie up your feet to prevent you from going to the lobby.
3. Ask her to tie up your arms with an unescapable item such as an armbinder.
4. Finally, ask her to gag you. Then you are unable to instruct her to change your binding to an escapable one.

Bravo! Restart the game and wait for a maid ~~or `InventoryRemove(Player, "ItemArms")`~~ to free yourself.

- - -

@GIT4NICK

**Comment by Ben987** [comment]
Thanks for the report!  I will send it over to git4nick.
